public/js/cita.json

```diff
@@ -5008,5 +4868,305 @@
+  },
+  {
+    "title": "Ribble Valley Homes",
+    "address": "De Lacy House\nStation Road\nClitheroe\nLancashire BB7 2JT",
+    "phone": null,
+    "hours": null,
+    "lat": 53.8721888,
+    "lng": -2.3941722,
+    "id": "004a7a00-ea5e-4972-9464-a441698a473e",
+    "booking_location_id": "c554946e-7b79-4446-b2cd-d930f668e54b"
+  },
+  {
+    "title": "Preston County Hall",
+    "address": "Fishergate\nPreston\nLancashire\nPR1 8XJ",
+    "phone": null,
+    "hours": null,
+    "lat": 53.7576414,
+    "lng": -2.7041301,
+    "id": "2ad27e4a-4ade-4b56-9345-bb8be98f6120",
+    "booking_location_id": "c554946e-7b79-4446-b2cd-d930f668e54b"
+  },
+  {
+    "title": "Rothwell One Stop Centre",
+    "address": "Marsh Street\nRothwell\nLS26 0AD",
+    "phone": null,
+    "hours": null,
+    "lat": 53.7471108,
+    "lng": -1.4783155,
+    "id": "8d672421-08e4-48fb-bc6c-b51f340b9e95",
+    "booking_location_id": "91b67ee6-f131-44d1-a05e-aac1306adba4"
+  },
+  {
+    "title": "Dewsbury Road One Stop Centre",
+    "address": "190 Dewesbury Road\nLeeds\nLS11 6PG",
+    "phone": null,
+    "hours": null,
+    "lat": 53.778261,
+    "lng": -1.5449519,
+    "id": "dd17d33a-f20a-474a-8d21-5d59b93eb3aa",
+    "booking_location_id": "91b67ee6-f131-44d1-a05e-aac1306adba4"
+  },
+  {
+    "title": "Cardiff Central Hub",
+    "address": "Advice Hub\nThe Hayes\nCardiff\nCF10 1FL",
+    "phone": null,
+    "hours": null,
+    "lat": 51.4785591,
+    "lng": -3.1757673,
+    "id": "edde9cde-41ac-4485-9919-b6d34d79f844",
+    "booking_location_id": "525da418-ff2c-4522-90a9-bc70ba4ca78b"
+  },
+  {
+    "title": "Llandaff North Library",
+    "address": "Gabalfa Avenue\nCardiff\nCF14 2HU",
+    "phone": null,
+    "hours": null,
+    "lat": 51.5026589,
+    "lng": -3.2230683,
+    "id": "2ec343f7-584b-4086-b5c3-e9c4b77922f7",
+    "booking_location_id": "525da418-ff2c-4522-90a9-bc70ba4ca78b"
+  },
+  {
+    "title": "Ammanford Library",
+    "address": "3 Wind Street\nAmmanford\nSA18 3DN",
+    "phone": null,
+    "hours": null,
+    "lat": 51.7927668,
+    "lng": -3.9884448,
+    "id": "36ab7bd7-87e2-47d6-8ffc-edd7e5264380",
+    "booking_location_id": "18ef0b7d-b13a-4284-9765-aa888aa4ba09"
+  },
+  {
+    "title": "Citizens Advice Nottingham and District",
+    "address": "Housing Advice Service\nArcher House\nCastle Gate (Access by Hounds Gate)\nNottingham\nNG1 7AW",
+    "phone": null,
+    "hours": null,
+    "lat": 52.9519347,
+    "lng": -1.1499251,
+    "id": "2e9f069a-b4b6-4b4a-a4a0-8ddd1516b064",
+    "booking_location_id": "b63e0799-2161-482f-bdfe-5188f904da32"
+  },
+  {
+    "title": "Chesterfield Library",
+    "address": "New Beetwell Street\nChesterfield\nDerbyshire\nS40 1QN",
+    "phone": null,
+    "hours": null,
+    "lat": 53.2343316,
+    "lng": -1.4281664,
+    "id": "5468cd16-a031-43a7-868a-77408a458c97",
+    "booking_location_id": "b63e0799-2161-482f-bdfe-5188f904da32"
+  },
+  {
+    "title": "Citizens Advice Leicestershire (Wigston)",
```

Comment:
We shouldn't include the name of the organisation in the title of the location. In research we saw this was decisive with our users. When the location is displayed on the Pension Wise website we explain the relationship with Citizens Advice.

As it stands it would read:

```
Your appointment will be at Citizens Advice Leicestershire (Wigston) Citizens Advice with a trained Pension Wise guidance specialist.
```

I would suggest a title of "Wigston" so that it reads:

```
Your appointment will be at Wigston Citizens Advice with a trained Pension Wise guidance specialist.
```